Dockerfile

```diff
@@ -0,0 +1,40 @@
+FROM ruby:2.1-slim
```

Comment:
Any strong reason why you're using the slim variant (the [Ruby Docker page](https://registry.hub.docker.com/_/ruby/) seems to discourage this)? I can even imagine that some `apt-get install`-ed packages are already already present when using the normal image, speeding up the process and using less space when many images are present. Haven't measured it, though.